client/src/components/SubmissionAudit.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <i18n-t
```

Comment:
Nice work with the il8n on this it's concise and easily understandable.👍

Reply:
I would attribute at least half of that success to your assistance on Monday.